parboiled-core/src/test/scala/org/parboiled2/ReductionTypeSpec.scala

```diff
@@ -26,12 +26,12 @@ object ReductionTypeSpec extends TestSuite {
   case class Foo2(lhs: Foo, rhs: Foo) extends Foo
 
   class FooParser(val input: ParserInput) extends Parser {
-    def OneOrMoreExpr  = rule(foo1 ~ oneOrMore(foo1 ~> Foo2))
```

Comment:
Wow, these tests look so benign, but it's quite mind-bending that you can build recursive structures like that...